tests/load/scripts/scramble_data.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import argparse
+import json
+
+from ..scrambler import json as json_scrambler
+
+
+def main() -> None:
+    """
+    Main entry point for the script.
+    """
+    parser = argparse.ArgumentParser(description="Scramble test data")
+    parser.add_argument("--file", type=str, help="The file to load the test data from")
+    args = parser.parse_args()
+
+    if args.file.endswith(".json"):
+        with open(args.file, "r") as f:
+            data = json.load(f)
```

Comment:
This is another place we'll have to keep an eye on.  I don't know if the plan was to load a file with 1.5 million records here, but if so, we might run into memory issues.

Reply:
Definitely!